_internal/utils/env.ts

```diff
@@ -6,7 +6,9 @@ export const IS_REACT_LEGACY = !React.useId
 export const IS_SERVER = !isWindowDefined || 'Deno' in window
 
 // Polyfill requestAnimationFrame
-export const rAF = (f: (...args: any[]) => void) =>
+export const rAF = (
+  f: (...args: any[]) => void
+): number | ReturnType<typeof setTimeout> =>
```

Comment:
```suggestion
export const rAF: (f: (...args: any[]) => any) => any = (f) =>
  hasRequestAnimationFrame()
    ? window['requestAnimationFrame'](f)
    : setTimeout(f, 1)
```
Since it's just a internal function. Let's keep it simple.

Reply:
it doesn't hurt to give the explict type, and I feel it's simpler